matrix_70X.txt

```diff
@@ -16,17 +16,27 @@ TTbarwf25p0 25.0_TTbar+TTbarINPUT+DIGI+RECOAlCaEle+HARVEST+ALCATT/step3.root REC
 TTbarwf25p0 25.0_TTbar+TTbar+DIGI+RECOAlCaEle+HARVEST+ALCATT/step3.root RECO 
 TTbarwf25p0 25.0_TTbar+TTbar+DIGI+RECOAlCaCalo+HARVEST+ALCATT/step3.root RECO
 TTbarwf25p0 25.0_TTbar+TTbarINPUT+DIGI+RECOAlCaCalo+HARVEST+ALCATT/step3.root RECO
-RunMinBias2011Awf4p17 4.17_RunMinBias2011A+RunMinBias2011A+HLTD+RECODreHLT+HARVESTDreHLT+SKIMDreHLT/step3.root reRECO
+RunMinBias2011Awf4p17 4.17_RunMinBias2011A+RunMinBias2011A+HLTD+RECODreHLT+HARVESTDreHLT+SKIMDreHLT/step3.root reRECO 4.17_RunMinBias2011A+RunMinBias2011A+HLTD+RECODreHLT+HARVESTDreHLT+SKIMDreHLT/step3_inMINIAOD.root reRECO
```

Comment:
let's enforce the same directory and change the format slightly:
"shortName fullDirName recoFileName recoProcessName miniAODFileName miniAODProcessName"